### ISSUE TITLE
Tests fail on v0.5

### ISSUE THREAD

**Issue by ranjanan** [FRBNY-DSGE/DSGE.jl#5]
```
ERROR: LoadError: InitError: type Method has no field func
 in macro expansion at /Users/ranjan/.julia/v0.5/Requires/src/require.jl:9 [inlined]
 in (::Requires.##8#9)() at /Users/ranjan/.julia/v0.5/Requires/src/init.jl:17
 in __init__() at /Users/ranjan/.julia/v0.5/Requires/src/init.jl:24
 in _require_from_serialized(::Int64, ::Symbol, ::String, ::Bool) at ./loading.jl:174
 in _require_from_serialized(::Int64, ::Symbol, ::Bool) at ./loading.jl:202
 in require(::Symbol) at ./loading.jl:332
 in stale_cachefile(::String, ::String) at ./loading.jl:562
 in recompile_stale(::Symbol, ::String) at ./loading.jl:579
 in _require_from_serialized(::Int64, ::Symbol, ::String, ::Bool) at ./loading.jl:173
 in _require_from_serialized(::Int64, ::Symbol, ::Bool) at ./loading.jl:202
 in require(::Symbol) at ./loading.jl:332
 in stale_cachefile(::String, ::String) at ./loading.jl:562
 in recompile_stale(::Symbol, ::String) at ./loading.jl:579
 in _require_from_serialized(::Int64, ::Symbol, ::String, ::Bool) at ./loading.jl:173
 in _require_from_serialized(::Int64, ::Symbol, ::Bool) at ./loading.jl:202
 in require(::Symbol) at ./loading.jl:332
 in include_from_node1(::String) at ./loading.jl:426
 in process_options(::Base.JLOptions) at ./client.jl:262
 in _start() at ./client.jl:318
during initialization of module Requires
while loading /Users/ranjan/.julia/v0.5/DSGE/test/runtests.jl, in expression starting on line 2
```

**Comment by ranjanan** [comment]
Are you guys using the `Requires` package anywhere? That package fails on v0.5

**Comment by emoszkowski** [comment]
Hey @ranjanan!  Thanks for the heads-up :) We're planning on fixing all bugs related to v0.5 when it becomes the official stable release (looks like it hasn't yet). 

Hope you enjoyed the rest of your time in the states! I know I owe you an email.

**Comment by ranjanan** [comment]
Hey @emoszkowski! :) 

>We're planning on fixing all bugs related to v0.5 when it becomes the official stable release (looks like it hasn't yet)

I posted this issue in light of the upcoming v0.5 release. This package [currently fails on PackageEvaluator](http://pkg.julialang.org/detail/DSGE.html), so it would be nice to fix this before the release. [A feature freeze has already been announced](https://github.com/JuliaLang/julia/pull/17503) and a release candidate is right around the corner. 

>Hope you enjoyed the rest of your time in the states! I know I owe you an email.

I did, thanks for asking! We'll catch up via email.
